.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a misbehaviour in the system
+
+---
+
+### Problem description
+
+<!-- Put here a concise description of the problem -->
+
+### Expected behaviour
+
+<!-- Write down the behaviour you would have expected -->
+
+### Actual behaviour
+
+<!-- Write down the actual behaviour of the system -->
+
+### Reproduction steps (if any)
```

Comment:
Can we encourage providing failing tests/scriptss?

Reply:
Yup -- I would put that into the "optional" category. What I am trying to strike is a good balance between "in house" bug reporting vs external bug reporting. For the former, we definitely want to keep this light and simple, for the latter, the more info the better.

I think that practically speaking the solution is to have our templates as comprehensive as possible, and when we open "in-house" reports, we strip and keep just the essential.